openlibrary/core/lending.py

```diff
@@ -311,6 +311,8 @@ def update_availability_schema_to_v2(v1_resp, ocaid):
         # endpoint
         v1_resp['is_browseable'] = (v1_resp['is_lendable'] and
                                     v1_resp['status'] == 'error')
+
+        v1_resp['data_src'] = 'core.models.lending.get_availability'
```

Comment:
Not used (and kind of defeats the point of having an abstraction; downstream clients shouldn't care who generated this dict. It should behave the same in all circumstances.

```suggestion
```